updater/src/Updater/Generate/Template.purs

```diff
@@ -154,7 +154,7 @@ runTemplate opts template = do
 filterByType :: { usesJS :: Boolean, templates :: Array TemplateSource } -> Array TemplateSource
 filterByType { usesJS: true, templates } = templates
 filterByType { usesJS: false, templates } =
-  filter (not <<< (JS == _) <<< _.sourceType) templates
+  filter (not <<< eq JS <<< _.sourceType) templates
```

Comment:
If you want to be _really_ fancy I believe you can just do `not eq JS <<< _.sourceType` ;)